test/outband/issue121.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-octal */
+// vim: expandtab:ts=2:sw=2
+
+var
```

Comment:
We should use `const` instead of `var`.

Reply:
There are multiple such cases. Let's do that prior to 0.1.0 release, what say ye?

Reply:
Back to normal again: this will all be cleaned up with #161 where I will fix the existing issues with async tests.

Reply:
done.